js-array-methods_for-each/js/index.js

```diff
@@ -19,4 +19,18 @@ const colors = [
   "#A5A5A5",
 ];
 
-colors.forEach();
+colors.forEach((color) => {
+  const colorElement = document.createElement("div");
+  colorElement.style.backgroundColor = color;
+  colorElement.classList.add("color-box");
+  document.body.append(colorElement);
+});
+
+function renderColorBox() {
```

Comment:
i think you're missing the parameter in this case color which you are referring to in line 31.
function renderColorBox(color) { ...